app/js/controllers/settings.js

```diff
@@ -29,27 +47,32 @@ export default class SettingsController extends Controller {
     });
 
     activity.onerror = (e) => {
-      console.error('Error opening Settings Add-ons panel', e);
+      console.error('Error opening "Settings > Add-ons" panel', e);
     };
   }
 
-  uninstall(addon) {
-    AddonService.uninstall(addon.origin).then(() => {
-      AddonService.getAddons(window.location.host).then((addons) => {
-        this.view.setAddons(addons);
-      });
+  showAddon(manifestURL) {
+    var activity = new window.MozActivity({
```

Comment:
yeah, you're right.. i think this was left over from when i was trying to diagnose the activity issues with addons